config/ab_tests.yml

```diff
@@ -29,15 +29,19 @@ shared:
     bar: 1
     baz: 1
   2021_02_cookie_consent_test:
-    none: 1
+    none: 0
     default: 1
-    top: 1
-    bottom: 1
-    modal: 1
-    gds: 1
+    top: 0
```

Comment:
Is this supposed to be here? This will prevent new sessions being assigned these variants, but it won't stop people seeing these variants if they already have it assigned in their session

Reply:
Ah ok. I thought this was all that was necessary to 'switch off' the A:B test. 

What can we do to switch off the A:B test for users who already have it assigned in their session?

Remove all variants, except the default, from the templates?

Reply:
To switch off the test we need to remove the code - this should be done in a separate PR